### PR TITLE
feat: add vite config; update yarn;

### PR DESCRIPTION
**Pull request by Lykhoyda**

### CODE REVIEW

File: tsconfig.json
Comment (BeroBurny) [inline tsconfig.json:1]: what is the rationale for adding `jsx`, `isolatedModules`, `types`, and `useDefineForClassFields`?

Comment (Lykhoyda) [reply tsconfig.json:1]: You can find all the info about typescript configuration with vite here https://vitejs.dev/guide/features.html#typescript-compiler-options

The "jsx" option is debatable, as react js is the most commonly used framework it probably makes sense to have preconfigured it by default. But we can keep it with default value as well

Comment (BeroBurny) [reply tsconfig.json:1]: I meant rationale for this project, as there is no usage of vite, jsx, transpiring ts to js

**Comment by mpetrunic** [comment]
added easier testing in #11

**Review by mpetrunic — CHANGES_REQUESTED** [review]
Please just add vite eslint rules, no need to modify projects tsconfig^^

**Pull request closed without merging**